.github/agents/README.md

```
@@ -39,59 +39,59 @@ The agent will:
-- Creates commits and pull requests for releases
+- Uses the canonical shared skill in `.agents/skills/release/`
+- Updates version files and changelogs for the operator, Helm chart, and CRDs chart when needed
+- Runs Helm snapshot updates, Helm tests, and manifest bundle regeneration
+- Leaves commit, push, and PR creation to the user unless explicitly requested
 
 **How to use it:**
 
-When working in a GitHub Copilot-enabled environment (VS Code, JetBrains, etc.), you can simply ask:
+When working in a GitHub Copilot-enabled environment, ask:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)